src/components/navbar.component.tsx

```diff
@@ -78,6 +78,22 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
               <Logo />
             </HeaderLink>
             <HeaderGlobalBar>
+              <ExtensionSlot extensionSlotName="add-patient-slot">
+                <HeaderGlobalAction
+                  aria-label="Add"
+                  aria-labelledby="Add Patient"
+                  name="AddPatientIcon"
+                >
+                  <Extension />
```

Comment:
You don't need this. Just write `<ExtensionSlot ... />` (self close = no children) and you are good.